### DISCUSSION TITLE
[Feature] User last active timestamp

### DISCUSSION THREAD

**Discussion by yetanotherusrname** [immich-app/immich#5166] [Feature Request]
### The feature

It would be nice to see the last time a user has been active on the server stats page with a column that shows last upload date. That way I can check to make sure users haven't screwed something up on their phone (or the OS killed tasks). This would give me a quick way to make sure that everything is still getting backed up correctly for all my users. 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by m-dubois** [comment]
Same need.
I searched on the API side to fulfill this need but did not find anything. Either way would be ok for me, but something in the admin UI is nice also :)

**Comment by lukyjay** [comment]
A new column on the user listing would be fantastic. E.g.:

![image](https://github.com/user-attachments/assets/9e3d7229-fe7e-4ac5-8e60-db6585e69f14)
